readme.md

```diff
@@ -58,6 +58,11 @@ node’s [`index`][index] and its [`parent`][parent].
 
 Otherwise the same as [`unist-util-visit-parents`][vp].
 
+See [`next? = visitor(node, ancestors)`][vp-visitor] in the
+[`unist-util-visit-parents`][vp] README for more documentation on traversing
+a tree with a `visitor`, including how to skip traversing descendants of a
+node, how to exit early, and how to traverse a tree while modifying it.
+
```

Comment:
lines 61-65 seem to be a circuitous repeat of line 59?
That links to the same readme as line 59 does?

Reply:
https://github.com/syntax-tree/unist-util-visit/issues/20#issuecomment-745413758

Reply:
Yeah, appreciate the PR, and thought on making it more clear. But this now to me reads as a duplication.
As a user, I’d think when I see three links that they’re going to different places, but in this case they’re all (roughly) the same.

Reply:
I agree on the duplication and the confusing links. I updated my PR. It has another link but it's just because it has another occurrence of `unist-util-visit-parents`.